Formula/corsixth.rb

```diff
@@ -46,23 +46,19 @@ def install
     ENV["FULL_PRODUCT_NAME"] = "CorsixTH.app"
 
     luapath = libexec/"vendor"
-    ENV["LUA_PATH"] = "#{luapath}/share/lua/5.2/?.lua"
-    ENV["LUA_CPATH"] = "#{luapath}/lib/lua/5.2/?.so"
+    ENV["LUA_PATH"] = "#{luapath}/share/lua/5.3/?.lua"
+    ENV["LUA_CPATH"] = "#{luapath}/lib/lua/5.3/?.so"
 
     resources.each do |r|
       r.stage do
         system "luarocks", "build", r.name, "--tree=#{luapath}"
       end
     end
 
-    # Ensures it uses the intended Lua (5.2) rather than 5.1/5.3 or
```

Comment:
Was this double checked locally on a system that has multiple versions of Lua installed?

Reply:
(CMake being CMake has the tendency to do whatever the hell it likes, even when passed direct variables in some cases, because life is fun 🙃)

Reply:
Yeah.